### ISSUE TITLE
Pagination - Ensure custom controls provide proper textual name, role, and state information - (2036570530)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#5590]
### Module: 
 22b Pagination 

### Violation: 
 Ensure custom controls provide proper textual name, role, and state information 

![image](https://user-images.githubusercontent.com/5023024/202749227-8faf69f7-2ddc-4775-894c-39dea53e09ca.png)

### WCAG Reference: 
- [1.1.1 Non-text Content](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html)
- [4.1.2 Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html) 

### Instance ID: 
 2036570530 

### Severity: 
 7 

### Description: 
 [Issue]
The current state information of the "pagination component" is not provided.

[User Impact]
Screen reader users will have difficulty determining the purpose and state of these controls.

[Code Reference] 
 ```
<button class="page is-selected">58</button>
 ``` 

### Note: 
 [Suggestion]
Ensure custom controls provide proper textual name, role, and state information. Properly structured native controls provide this information automatically. For custom controls, developers must explicitly add this information by using ARIA attributes.

Use the aria-current="page" class to highlight the currently active page.

[Compliant Code Example] 
 ```
<button class="page is-selected" aria-current="page">58</button>
 ```
 

### Media Type: 
 Custom Controls 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by brittneytewks** [comment]
Removed "needs triage" since a priority is set and estimate is indicated

**Comment by geospatialem** [comment]
Verified on `master` with NVDA and JAWS.
